client/foundries_el2g.go

```diff
@@ -106,33 +106,35 @@ func (a *Api) El2gDevices(factory string) ([]El2gDevice, error) {
 	return devices, nil
 }
 
-func (a *Api) El2gAddDevice(factory, prodId, deviceUuid string) error {
+func (a *Api) El2gAddDevice(factory, prodId, deviceUuid string, production bool) error {
 	url := a.serverUrl + "/ota/factories/" + factory + "/el2g/devices/"
 	devices := []string{deviceUuid}
 
 	type Req struct {
-		ProductId string   `json:"product-id"`
-		Devices   []string `json:"devices"`
+		ProductId  string   `json:"product-id"`
+		Devices    []string `json:"devices"`
+		Production bool     `json:"production"`
 	}
 
-	body, err := json.Marshal(Req{prodId, devices})
+	body, err := json.Marshal(Req{prodId, devices, production})
 	if err != nil {
 		return err
 	}
 	_, err = a.Post(url, body)
 	return err
 }
 
-func (a *Api) El2gDelDevice(factory, prodId, deviceUuid string) error {
+func (a *Api) El2gDelDevice(factory, prodId, deviceUuid string, production bool) error {
```

Comment:
Why do we need to pass the `production` flag into the `delete` operation?

Reply:
its a different device group in their product. w/o that flag i don't know where to find the device i need to delete. We can try to hide it. then you make 3 api calls, "find this device here? find it here?, okay delete that one".

Reply:
- On the back-end when we fetch the device from the database, doesn't we know if it is production or not?
- Can the non-production device belong to the production El2Go product?

I feel like I'm definitely missing something here.

Reply:
this isn't our database. these commands are for managing the device inside el2g's product. The device might not even exist yet in our product.

Reply:
yay, that's more complex than I thought... just ignore it.